src/bindings/c/tests/ov_test.hpp

```diff
@@ -25,22 +26,30 @@
 extern std::map<ov_element_type_e, size_t> element_type_size_map;
 #define GET_ELEMENT_TYPE_SIZE(a) element_type_size_map[a]
 
+inline std::string get_extension_file_path();
+
 class ov_capi_test_base : public ::testing::TestWithParam<std::string> {
 public:
     void SetUp() override {
         TestDataHelpers::generate_test_model();
         xml_file_name = TestDataHelpers::get_model_xml_file_name();
         bin_file_name = TestDataHelpers::get_model_bin_file_name();
+        extension_file_path = get_extension_file_path();
     }
 
     void TearDown() override {
         TestDataHelpers::release_test_model();
     }
 
 public:
-    std::string xml_file_name, bin_file_name;
+    std::string xml_file_name, bin_file_name, extension_file_path;
 };
 
+inline std::string get_extension_file_path() {
```

Comment:
Move inside the test fixture `ov_capi_test_base `?

Reply:
Done, moved inside `ov_capi_test_base`